x-pack/metricbeat/module/googlecloud/stackdriver/_meta/docs.asciidoc

```diff
@@ -12,8 +22,12 @@ type. Metric type is to used for identifying a specific time series.
 
 * *aligner*: A single string with which aggregation operation need to be applied
 onto time series data for ListTimeSeries API. If it's not given, default aligner
-is set to be `ALIGN_NONE`. Sample period of each metric type is obtained from
-making https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors/list [ListMetricDescriptors API] call.
+is set to be `ALIGN_NONE`. Google Cloud also supports `ALIGN_DELTA`, `ALIGN_RATE`,
+`ALIGN_MIN`, `ALIGN_MAX`, `ALIGN_MEAN`, `ALIGN_COUNT`, `ALIGN_SUM` and etc.
+Please see
+https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.alertPolicies#aligner[Aggregation Aligner]
```

Comment:
There's also this: https://cloud.google.com/monitoring/api/ref_v3/rpc/google.monitoring.v3#aligner
What would be the correct link here?

Reply:
Ahh your link is the right one. I guess alert policies have aligner too in gcp 😂 Thanks!